update_unix.go

```diff
@@ -40,15 +40,24 @@ func RunUpdate() bool {
 func fetchUpdateScript() (string, error) {
 	resp, err := http.Get("https://raw.githubusercontent.com/coveo/tgf/master/get-latest-tgf.sh")
 	if err != nil {
-		Print("Error fetching update script", err)
+		log.Fatal("Error fetching update script", err)
```

Comment:
I think these errors should be a warning and cancel the update. But TGF should still run